### PR TITLE
[15.0] [MIG] webservice

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration
@ForgeFlow

### CODE REVIEW

File: webservice/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "WebService",
+    "summary": """
+        Defines webservice abstract definition to be used generally""",
+    "version": "15.0.1.0.0",
+    "license": "AGPL-3",
+    "development_status": "Alpha",
```
Comment (LoisRForgeFlow) [inline webservice/__manifest__.py]: Can you change this to Beta at least?

@etobella maybe more?

Comment (etobella) [reply webservice/__manifest__.py]: I think we can put as Mature and add me as maintainer WDYT?

Comment (LoisRForgeFlow) [reply webservice/__manifest__.py]: I agree, it is done!

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by etobella — APPROVED** [review]

**Comment by simahawk** [comment]
FYI being moved to web-api here https://github.com/OCA/edi/pull/641

**Pull request merged**